src/django/api/views.py

```diff
@@ -878,8 +878,8 @@ def get_serializer_fields(self, path, method):
                     name='data',
                     location='body',
                     description=(
-                        'The country, name, and address of the facility. See '
-                        'the sample request body above.'),
+                        'The country, name, address and sector(s) of the '
+                        'facility. See the sample request body above.'),
```

Comment:
@jwalgran this accidentally got rebased into this commit before the review was done - just wanted to flag that this did get updated